### PR TITLE
[KG][Score Func] Update TransE with L2 distance support.

### PR DESCRIPTION
**Pull request by classicsong**
Now TransE support both l1 distance and l2 distance for calculating scores.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [x] Related issue is referred in this PR

### CODE REVIEW

File: apps/kg/models/mxnet/score_fun.py [hunk 1]
```diff
@@ -5,16 +5,20 @@
 from mxnet import ndarray as nd
 
 class TransEScore(nn.Block):
-    def __init__(self, gamma):
+    def __init__(self, gamma, dist_func='l2'):
```
Comment (zheng-da) [inline apps/kg/models/mxnet/score_fun.py]: can you make the same modification for MXNet?

Comment (classicsong) [reply apps/kg/models/mxnet/score_fun.py]: You means supporting l2 distance for transE? The PR already has it.

Comment (zheng-da) [reply apps/kg/models/mxnet/score_fun.py]: i mean, can we have the similar implementation as the one in Pytorch version? so mxnet version and pytorch version has similar speed.

Comment (classicsong) [reply apps/kg/models/mxnet/score_fun.py]: > i mean, can we have the similar implementation as the one in Pytorch version? so mxnet version and pytorch version has similar speed.

Done.

**Pull request merged**